tests/unittests/onnxImporterTest.cpp

```diff
@@ -612,17 +612,18 @@ TEST(onnx, expandDims) {
 TEST(onnx, gather) {
   ExecutionEngine EE;
   auto &mod = EE.getModule();
-  auto *F = mod.createFunction("main");
   std::string netFilename("tests/models/onnxModels/gather.onnxtxt");
+  auto *F = mod.createFunction("main");
   Placeholder *output;
   Tensor data(ElemKind::FloatTy, {3, 2});
-  Tensor indices(ElemKind::FloatTy, {2, 4});
+  Tensor indices(ElemKind::Int32ITy, {2, 4});
 
   {
     ONNXModelLoader onnxLD(netFilename, {"data", "indices"},
                            {&data.getType(), &indices.getType()}, *F);
     output = onnxLD.getSingleOutput();
   }
+  EE.compile(CompilationMode::Infer, F);
```

Comment:
no need to compile since no computations are done (operatorTest runs through the compilation anyway)

Reply:
Actually, I added this explicitly because previous one didn't really go through the verification. That's why the test could even pass with Indices of type `float`.